app/helpers/automated_tests_helper.rb

```diff
@@ -5,6 +5,14 @@ module AutomatedTestsHelper
   # it is enqueued and it is waiting for execution. Resque manages this queue.
   @queue = :test_waiting_list
 
+  def fetch_latest_tokens_for_grouping(grouping)
```

Comment:
Unused method argument - `grouping`. If it's necessary, use `_` or `_grouping` as an argument name to indicate that it won't be used. You can also write as `fetch_latest_tokens_for_grouping(*)` if you want the method to accept any arguments but don't care about them.